pkg/apis/provisioning/v1alpha3/provisioner.go

```diff
@@ -130,7 +133,7 @@ var (
 	InstanceTypeLabelKey = "node.kubernetes.io/instance-type"
 
 	// Finalizers
-	KarpenterFinalizer = SchemeGroupVersion.Group + "/termination"
+	TerminationFinalizer = SchemeGroupVersion.Group + "/termination"
```

Comment:
It's generally a best practice to avoid self referencing your project, since it's a nightmare if you ever want to rebrand. I see your point about knowing that it's karpenter specific, but I think this is already heavily implied by the fact that it's owned by the karpenterv1alpha3 package.

Reply:
Fair point.